website/src/pages/500.astro

```diff
@@ -11,18 +15,14 @@ interface Props {
 
 const { error } = Astro.props;
 
-getInstanceLogger('500ErrorPage').error(`An error occurred: ${getErrorLogMessage(error)}`);
+const errorId = uuidv4();
+getInstanceLogger('500ErrorPage').error(`An error occurred: ${getErrorLogMessage(error)}`, { errorId });
 ---
 
 <BaseLayout title='GenSpectrum Dashboards'>
     <PageContainer>
         <PageHeadline>Oops! An error occurred...</PageHeadline>
-        <div class='mb-2'>We are sorry for the inconvenience. Please try again.</div>
-        <div>
-            If the problem persists, please create an issue on <a
-                class='text-primary underline'
-                href='https://github.com/GenSpectrum/dashboards'>GitHub</a
-            >, describing the error.
-        </div>
+        <div class='mb-2'>We are sorry for the inconvenience. Please try again later.</div>
```

Comment:
```suggestion
        <div class='mb-2'>We are sorry for the inconvenience. Please try again later.</div>
        <div>You have seen this message multiple times?</div>
```

Reply:
I would put the `ErrorReportInstruction inside a collapse and make the "You have seen this message multiple times?" the handle.
This way the user is not exposed to too much information.